src/get-started/test-drive.md

```diff
@@ -186,5 +186,3 @@ You should see the updated string in the running app almost immediately.
 ## Next step
 
 Let's learn some core Flutter concepts, by creating a small app.
-
-[Next step: Write your first app](/get-started/codelab)
```

Comment:
@Sfshaza FYI, I dropped the hardcoded "Next step" links (since each Get Started page now has next/prev links at the top and bottom of the pages.